importscan/scan.py

```diff
@@ -89,6 +90,11 @@ def handle_error(name, e):
       ``handle_error`` does not re-raise the error, the error is
       suppressed.
     """
+    # module name instead of package object
+    if not hasattr(package, '__name__'):
```

Comment:
Could you explain why you want to support string names as an argument?

Reply:
@faassen simply to enable simple usage of `scan(__name__)` which is quite gruesome to write otherwise